pages/partials/mathjax.ejs

```diff
@@ -1,7 +1,15 @@
 <script>
 
 // default to CHTML because of https://github.com/PrairieLearn/PrairieLearn/pull/3520
-var outputComponent = 'output/chtml';
+// var outputComponent = 'output/chtml';
+
+// Default to SVG again: Windows+Chrome was the primary case where the CHTML
+// mode was active, but now thin lines are disappearing on that platform too.
+var outputComponent = 'output/svg';
+
+// With 'output/svg' now being the default, the following checks may be
+// redundant, but they could also be useful reference if new issues seem to
+// arise.
```

Comment:
Let's just remove all this; it will pretty much immediately become debt. We have `git blame` and history if we ever want to know what this looked like previously.